src/ima.js

```diff
@@ -561,14 +561,33 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
     this._sdk.settings.setPlayerType(this.config.playerName);
     this._sdk.settings.setPlayerVersion(this.config.playerVersion);
     this._sdk.settings.setVpaidAllowed(true);
-    this._sdk.settings.setVpaidMode(this._sdk.ImaSdkSettings.VpaidMode.ENABLED);
+    this._sdk.settings.setVpaidMode(this._setVpaid());
     if (typeof this.config.setDisableCustomPlaybackForIOS10Plus === 'boolean') {
       this._sdk.settings.setDisableCustomPlaybackForIOS10Plus(this.config.setDisableCustomPlaybackForIOS10Plus);
     } else {
       this._sdk.settings.setDisableCustomPlaybackForIOS10Plus(this.player.config.playback.playsinline);
     }
   }
 
+  /**
+   * Initializing the VpaidMode.
+   * @private
+   * @returns {void}
+   * @instance
+   * @memberof Ima
+   */
+  _setVpaid(): void {
```

Comment:
the method returns the vpaid mode so better name will be `_getVpaidMode`, also the method returns a string which means `VpaidModeEnumType`, so it's not void(both Flow type and JSDoc)